src/backend/controllers/users.js

```diff
@@ -0,0 +1,168 @@
+import { generateJWT, hashPassword} from '../utils/authorization.js';
+import model from "../model/users.js";
+
+export async function signup(req, res) {
+
+    if (!req.body.StudentID || !req.body.SchoolEmail || !req.body.Username || !req.body.Fname || !req.body.Lname || !req.body.Password) {
+        return res.status(400).send('Missing value');
```

Comment:
Suggest putting all error messages in { error: "message" } instead of directly putting it in the body. Same goes for all other error messages below and in other files

Reply:
OK